.github/tests/test_model_regression_test_read_dataset_branch_tmpl.py

```diff
@@ -0,0 +1,22 @@
+import pathlib
+import subprocess
+import pytest
+
+TEMPLATE_FPATH = ".github/templates/model_regression_test_read_dataset_branch.tmpl"
+REPO_DIR = pathlib.Path("").absolute()
+TEST_DATA_DIR = str(pathlib.Path(__file__).parent / 'test_data')
+DEFAULT_DATASET_BRANCH = "main"
+
+
+@pytest.mark.parametrize(["comment_body_file", "expected_dataset_branch"],
```

Comment:
I noticed that many of our unit tests have this format

```suggestion
@pytest.mark.parametrize("comment_body_file,expected_dataset_branch",
```